share/urlwatch/examples/hooks.py.example

```diff
@@ -41,7 +41,10 @@ from urlwatch import reporters
 #    __required__ = ('username', 'password')
 #
 #    def retrieve(self, job_state):
-#        return 'Would log in to {} with {} and {}\n'.format(self.url, self.username, self.password)
+#        return (
+#            'Would log in to {} with {} and {}\n'.format(self.url, self.username, self.password),
```

Comment:
Unrelated change, please remove.

Reply:
This pull request changes `JobState.process()` so that it requires all `retrieve` implementations to return a tuple:
```python
  data, content_type = self.job.retrieve(self)
```
So this example needs updating unless backward compatibility is implemented in `JobState.process()`.

Reply:
Oh yeah true.. That's kind of unfortunate, it would by default break all custom user scripts. I would make it so that `self.job.retrieve(self)` can handle both the return value being a string (legacy behavior) or the return value being a 2-tuple. But let's think a bit more about this (I'll add another PR review comment).